src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Applicant.java

```diff
@@ -183,6 +183,15 @@ public boolean isBasedOverseas() {
             && !("United Kingdom").equalsIgnoreCase(homeAddress.getCountry());
     }
 
+    @JsonIgnore
+    public String getCorrespondenceEmail() {
+        if (isRepresented()) {
```

Comment:
My personal preference is for ternary if statements, but that is just a personal preference